workflows/scRNAseq/baredsc/README.md

```diff
@@ -0,0 +1,29 @@
+# BaredSC Workflows
+
+These workflows allow to run a baredSC analysis in a single click. It uses models from 1 to N Gaussians and combine them. It uses the logNorm scale, 100 bins for 1 dimension and 25 bins on each axis in 2 dimensions.
+
+## Inputs dataset
+
+- Both workflows need a tabular dataset where each row is a cell. The tabular needs to have a header line with column names. There must be at least two columns: 'nCount_RNA' and another one with the counts for the gene(s) of interest.
```

Comment:
Where would I find / how would I produce such a file ?

Reply:
Yes I should add it into the README. I will do it tomorrow.